packages/matter.js/src/time/Time.ts

```diff
@@ -24,6 +26,24 @@ export abstract class Time {
     static readonly getPeriodicTimer = (intervalMs: number, callback: TimerCallback): Timer => Time.get().getPeriodicTimer(intervalMs, callback);
 }
 
+const DefaultTime = new class extends Time {
```

Comment:
is that needed with your below "setup.ts" for all generator stuff?

Reply:
There are other places where it's needed, yeah, but I could patch those up too.  I actually meant to take it out of setup.ts.  After our last conversation it didn't sound like there was a good argument for having a broken default time implementation?

Reply:
We are at the browser compatibility again ... sooo ... if new Date() is in the end fine for "these days current" browsers basically then we can do as you propose

Reply:
BTW: Alternatively we could do it differently: We could make sure that when logger gets included it fallsback to such a logger "default dat thingy" when nothing is set, so that onl logger would use this ... wdyt?

Reply:
Looks like the last browser that didn't support Date (MSFT of course) was replaced in 2015.  There'll be plenty of features we take for granted that'll fail before Date does (e.g. most of ES6 and much of ES5).  My vote is to make it just work by default.  Could add a test to avoid installing default implementation if Date isn't present but even that feels like unnecessary noise.

Reply:
ok then lets go that way ... this should be tested automatically already